src/test/java/hudson/plugins/throttleconcurrents/ThrottleJobPropertyTest.java

```diff
@@ -124,27 +139,21 @@ public void testDescriptorImpl_should_a_concurrency_safe_list_for_categories(){
 
 
     private void assertProjects(String category, AbstractProject<?,?>... projects) {
-        jenkins.setAuthorizationStrategy(new RejectAllAuthorizationStrategy());
+        // Queue runs as ANONYMOUS.
+        // Ensure throttle-concurrent-builds works even without any permissions for ANONYMOUS.
+        jenkins.setAuthorizationStrategy(new GlobalMatrixAuthorizationStrategy());
+        SecurityContext orig = SecurityContextHolder.getContext();
+        NotSerilizableSecurityContext auth = new NotSerilizableSecurityContext();
+        auth.setAuthentication(Jenkins.ANONYMOUS);
+        SecurityContextHolder.setContext(auth);
         try {
             assertEquals(new HashSet<Queue.Task>(Arrays.asList(projects)), new HashSet<Queue.Task>
                     (ThrottleJobProperty.getCategoryTasks(category)));
         } finally {
+            SecurityContextHolder.setContext(orig);
             jenkins.setAuthorizationStrategy(AuthorizationStrategy.UNSECURED); // do not check during e.g. rebuildDependencyGraph from delete
         }
     }
-    private static class RejectAllAuthorizationStrategy extends AuthorizationStrategy {
-        RejectAllAuthorizationStrategy() {}
-        @Override public ACL getRootACL() {
-            return new AuthorizationStrategy.Unsecured().getRootACL();
-        }
-        @Override public Collection<String> getGroups() {
-            return Collections.emptySet();
-        }
-        @Override public ACL getACL(Job<?,?> project) {
-            fail("not even supposed to be looking at " + project);
-            return super.getACL(project);
-        }
-    }
```

Comment:
Existing tests used custom `AuthorizationStrategy` to ensure `getACL` is not used.
This is not applicable to cloudbees-folder as it checks `READ` permissions.
Anyway, that's not actually what we want to test, but what we want to test is that throttle-consurrent-build works even as `ANONYMOUS` with no permissions.